### PR TITLE
Added some fields for KeymapManager

### PR DESCRIPTION
**Pull request by nstbayless**
`KeymapManager` now contains the actual keymap!

### CODE REVIEW

File: src/InputManager.hpp [hunk 1]
```diff
@@ -23,8 +23,29 @@ namespace SokuLib
 	//CInputManagerCluster
 	struct KeymapManager {
 		char unknown[0x4];
+		// isPlayer is -1 when using keyboard, 0 when using first gamepad, 1 when using second gamepad.
 		char isPlayer;
-		char unknown2[0x33];
+		char unknown2[0x3];
```
Comment (PinkySmile) [inline src/InputManager.hpp:28]: This is probably padding honestly, you can remove it; or the isPlayer field is a multibyte field

File: src/InputManager.hpp [hunk 2]
```diff
@@ -23,8 +23,29 @@ namespace SokuLib
...
+		union
```
Comment (PinkySmile) [inline src/InputManager.hpp:29]: I don't think the union is really worth it here, I think the "bindings" field is that necessary. Removing the struct + union would make it more readable probably. You can move the struct out if you think having "binding" in front of everything is weird, or make it a int[], indexed using an enum.

**Review by PinkySmile — CHANGES_REQUESTED** [review]
Thanks for the contribution ! Please make the PR to dev though as master is way out of date

**Comment by nstbayless** [comment]
Sounds good, I'll close this PR and open another one.

**Pull request closed without merging**